adapter/pkg/messaging/eventtypes.go

```diff
@@ -67,7 +67,7 @@ type Event struct {
 
 // APIEvent for struct API events
 type APIEvent struct {
-	APIID         string   `json:"apiId"`
+	APIID         int32   `json:"apiId"`
```

Comment:
This should be a UUID. Hence the type is string. I scanned through the code and noticed that the event usage of APIID is being parsed into string as well. Hence no point of keeping this as int.

Reply:
Then revert the changes in notificationListner.go

Reply:
Done